### PR TITLE
HCALDQM Updates upon the request from HCAL Operations/DPG

### PR DESCRIPTION
**Pull request by vkhristenko**
cherry-picked from 80X commits from 14742

There are no significant updates, all of them are either tweaking some thresholds or some requests of parameter tweaking:

1) Accomodate Monitoring QIE11/10 in the release
2) Raw Task use 16LS bits not 32bits for comparison of ORN between AMC13/uHTR
3) Use of Unconstrained timing reconstruction of HBHE in Online!
4) Additional module to monitor all the channels w/o the use of Quality Masks from Conditions
5) Tweaking Offline thresholds
6) Synchronization across all of HCALDQM devs.

V

### CODE REVIEW

**Comment by vkhristenko** [comment]
@cmsbuild please test

**Comment by vanbesien** [comment]
please test

File: DQM/HcalTasks/plugins/QIE10Task.cc [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+	//	finish
+	DQTask::endLuminosityBlock(lb, es);
+}
+
+/* virtual */ void QIE10Task::_process(edm::Event const& e, 
+	edm::EventSetup const&)
+{
+	edm::Handle<QIE10DigiCollection> cqie10;
+	if (!e.getByToken(_tokQIE10, cqie10))
+		std::cout << "Collection isn't available" << std::endl;
```
Comment (dmitrijus) [inline DQM/HcalTasks/plugins/QIE10Task.cc]: Please replace with LogWarning, LogError or cms::Exception.
Use of std::cout is forbidden!

**Comment by dmitrijus** [comment]
80x reference: https://github.com/cms-sw/cmssw/pull/14742

**Comment by dmitrijus** [comment]
+1

This should be merged. It has been open for far too long. 
No std::cout should be implemented in the next PR...

File: DQM/HcalTasks/plugins/QIE10Task.cc (hunk 1 above)
Comment (davidlange6) [inline DQM/HcalTasks/plugins/QIE10Task.cc]: @vkhristenko @dmitrijus - yes, this needs to be changed

File: DQM/HcalCommon/interface/Constants.h [hunk 2]
```diff
@@ -215,6 +215,42 @@ namespace hcaldqm
 		int const RAW_EMPTY = 16;
 		int const UTCA_DATAFLAVOR = 0x5;
 
+		double const adc2fC[256] = {
```
Comment (davidlange6) [inline DQM/HcalCommon/interface/Constants.h]: dqm doesn't seem like the place we should be defining adc2fC. I guess this is already done elsewhere?

**Comment by vkhristenko** [comment]
@davidlange6 this array is for QIE10. As Phase 1 is in dev stage, I've included that in here, but it's used only for Onliny DQM

**Comment by davidlange6** [comment]
we seem to have similar yet different arrays defined for qie10 elsewhere? no?

**Comment by vkhristenko** [comment]
@davidlange6 , these updates went into Online DQM quite a while ago and PR with this array has been submitted long time ago. Regarding if it's defined anyelse, I don't know. Since RECO has not been fully developed yet for Phase1, which is where this array will only be used, I included that in the release - but it will be removed once Phase soft is fully deved.

VK

**Comment by abdoulline** [comment]
Concerning adc2FC conversion for QIE10: 

(1) "coder" prepared (generic piece both for QIE8/QIE10,  as QIE8/QIE10 comes from HcalQIETypes in DB and defines QIEShape for DetId "behind the scene"):
http://cmslxr.fnal.gov/source/RecoLocalCalo/HcalRecProducers/src/HcalSimpleReconstructor.cc?v=CMSSW_8_1_0_pre8#0272

(2)
then used in Digi-> CaloSamples (fC) conversion
 http://cmslxr.fnal.gov/source/RecoLocalCalo/HcalRecAlgos/src/HcalSimpleRecAlgo.cc?v=CMSSW_8_1_0_pre8#0174

**Comment by vkhristenko** [comment]
a new one is preped #15380 closing

**Pull request closed without merging**